data-loading/src/test/java/no/sikt/nva/data/report/api/etl/SingleObjectDataLoaderTest.java

```diff
@@ -139,10 +154,29 @@ private static String constructCompressedFileIdentifier(UUID identifier) {
         return identifier.toString() + GZIP_ENDING;
     }
 
-    private PersistedResourceEvent setupExistingIndexDocumentAndCreateUpsertEvent(TestData testData)
+    private String getExpectedData(TestData testData, ReportType reportType) {
+        return switch (reportType) {
+            case AFFILIATION -> testData.getAffiliationResponseData();
+            case CONTRIBUTOR -> testData.getContributorResponseData();
+            case FUNDING -> testData.getFundingResponseData();
+            case IDENTIFIER -> testData.getIdentifierResponseData();
+            case PUBLICATION -> testData.getPublicationResponseData();
+            case NVI -> testData.getNviResponseData();
+        };
```

Comment:
Maybe add a default that throws Illegal argument or unknown something exception, to cover the case when we add some new report type.